### PR TITLE
[cypress]  A test to check kiali help->view debug info

### PR DESCRIPTION
**Pull request by skondkar**
Cypress: A cypress test to check kiali help->View Debug info

### CODE REVIEW

File: frontend/cypress/integration/featureFiles/kiali_help.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: Kiali help
+
+  User wants to login to Kiali and view kiali help like documentation, view bebug info, certificate information, about info
+
+  Scenario: Login to Kiali and Open Kiali home page
+    Given user opens base url
+    And user clicks Log In With OpenShift
+    And user clicks my_htpasswd_provider
+    And user fill in username and password
+    Then user see console in URL
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/kiali_help.feature:10]: please refactor this scenario with 
`Given user is at administrator perspective`
for correct usage see
https://github.com/kiali/kiali/blob/bd662370f0950963d1151825915fc9e91db39474/frontend/cypress/integration/featureFiles/kiali_cookie.feature#L6

File: frontend/cypress/integration/common/kiali_help.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    if (auth_strategy === 'openshift') {
+        cy.get('#inputUsername').type('' || USERNAME);
+        cy.get('#inputPassword').type('' || PASSWD);
+        cy.get('button[type="submit"]').click()
+    }
+})
+
+Then('user see console in URL', () => {
+    cy.url().should('include', 'console')
+})
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/common/kiali_help.ts:51]: please refactor this code. If you reuse `Given user is at administrator perspective` step correctly (see below), this verbose login code could be deleted. Don't forget to rebase.

**Review by ScriptingShrimp — CHANGES_REQUESTED** [review]
please address required changes, rebase and squash commits. :+1:

**Comment by lucasponce** [comment]
Is this PR going to progress? It's open for 1.5 months.

**Comment by lucasponce** [comment]
Closing. If there is activity on this PR, please re-open it, but I'll close old PRs that are 2 months without progress.

**Pull request closed without merging**